drizzlepac/runsinglehap.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python
+
+""" runsinglehap.py - Module to control processing of single-visit mosaics
+
+:License: :doc:`LICENSE`
+
+USAGE: runsinglehap [-d] inputFilename
+
+Python USAGE:
+    python
+    from drizzlepac import runsinglehap
+    runsinglehap.process(inputFilename,debug=True)
+
```

Comment:
Just a typo here: "process" should be "perform".

Reply:
Resolved in c569ae17